src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -78,4 +97,46 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           + " " + lineContent);
     }
   }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Maybe move content from `doGet()` into a specialized helper method. Need to make a similar change in my code but each method should be responsible for one 'task'

Reply:
I'll fix it in a later pull request if that's okay. I made an issue for it so I won't forget (#59).

Reply:
I ended up fixing it for this pull request.